chevah_build

```diff
@@ -342,10 +338,11 @@ case $OS in
         export CXX="clang++"
         export CFLAGS="$CFLAGS -mmacosx-version-min=10.8"
         export MACOSX_DEPLOYMENT_TARGET=10.8
-        # OS X 10.8 not supported by cryptography 1.9, so we use 1.8.2.
+        # OS X 10.8 not supported by cryptography 1.9, so you'll normally use
+        # 1.8.2, but cryptography 2.2.2 has a working wheel for MacOS 10.6+.
```

Comment:
Mabye we should mention why the wheel works.

I guess that it works as comes with a static OpenSSL?

If we have static OpenSSL in OSX and RHEL5/SLES11

Once this is used in server we need to update the documentation at https://www.sftpplus.com/documentation/sftpplus/latest/guides/cryptography.html#openssl-versions-available-in-most-popular-operating-systems

Does this make sense?

Reply:
True, those wheels for OSX and RHEL5/SLES11 come with static OpenSSL 1.1.0h, as mentioned in the PR. I will add a mention in the comments here as well.

Documentation should be updated, of course. It's already outdated for AIX and Solaris.